frequent_flyer/README.md

```diff
@@ -0,0 +1,61 @@
+# Gaia Frequent Flyer Example
+
+
+
+This example models a frequent flyer program.
+
+The schema tracks the following the information:
+
+- airport: Airport information.
+- airplane: Information about the airplane.
+- route: Information about route.
+- flight: Information about the flight. A flight is connected in a:
+  - 1:n relationship to travelers and segments.  
+  - 1:1 relationship with depart and arrival airports.
+- segment: Describes the trip segment. A segments is connected in a:
+  - 1:1 relationship to trips
+  - 1:1 relationship to flights
+- trip: Describes information about the itinerary for a single traveler. A trip is connected in a:
+  - 1:1 relationship to airport for arriving flights
+  - 1:1 relationship to airport for departing flights
+- traveler: Information about the traveler. Travelers are connected to trip
+
+The initial example defines three rules that use the traveler and flight tables
+
+After you understand these rules you can extend the sample here in the sandbox.
+
+## Rule 1
+
+Tracks lifetime miles for the traveler.
+
+- `flight_miles` from the flights table is defined as an [Active Field](https://gaia-platform.github.io/gaia-platform-docs.io/articles/rulesets-gaia-programming-model.html?q=active%20fields).
+- When the Frequent Flyer app updates the field in the database, the rules engine fires the rule.
```

Comment:
Should it be `Rules Engine`? I think that we were capitalizing the names of the main platform components.

I'm not sure if Don left some glossary behind to show what terms we should be using. That would have been helpful.